src/compose.ts

```diff
@@ -218,7 +215,10 @@ function normalizeKeyValuePairs(obj?: ListOrDict, sep: string = '='): Dict<strin
 			.value();
 	}
 	return _(obj)
-		.map(val => val.split(sep, 2))
+		.map(val => {
+			const parts = val.split(sep);
+			return [ parts.shift()!, parts.join('=') ];
+		})
```

Comment:
This fixes env var parsing.

Reply:
Can we be sure that the `parts.shift()` will return a value here? (I'm not too familiar with JSON schemas)

Reply:
Yeah, worst case is `parts.length === 1`, so `parts.shift()` will always return a value.